charts/acp/templates/ingress.yaml

```diff
@@ -70,7 +67,7 @@ metadata:
     {{- include "acp.labels" . | nindent 4 }}
   annotations:
     kubernetes.io/ingress.class: nginx
-    nginx.ingress.kubernetes.io/auth-tls-verify-client: "on"
+    nginx.ingress.kubernetes.io/auth-tls-verify-client: "optional_no_ca"
```

Comment:
let's do this instead

```
 nginx.ingress.kubernetes.io/server-snippet: |
      ssl_verify_client optional_no_ca;
      ssl_verify_depth 2;
```

the depth is required in some cases

Reply:
I added the depth annotation.